pages/common/mongoimport.md

```diff
@@ -0,0 +1,24 @@
+# mongoimport
+
+> Imports content from a JSON, CSV, or TSV file into a MongoDB.
```

Comment:
```suggestion
> Imports content from a JSON, CSV, or TSV file into a MongoDB database.
```

Reply:
the db stands for database as you know, not sure about this one since that would read, "Mongo database database

Reply:
I don't think that MongoDB is generally read as `Mongo database`.  I'm a seasoned mongo user...this is correct 😉

Reply:
I think about `instance` but I was expecting comments.

Reply:
Either work fine.  Which one do you prefer?